### DISCUSSION TITLE
Will there be a 2024 roadmap ?

### DISCUSSION THREAD

**Discussion by Laudenlaruto** [Azure/CloudShell#381] [General]
Hey,

I have started using this cloud shell on my project and have been wondering if there will be a 2024 update to the roadmap ?

**Comment by mbifeld** [comment]
Hi [Laudenlaruto](https://github.com/Laudenlaruto)!  Yes, we'll be working on updating the roadmap for 2024.  Essentially, the focus feature-wise will be on rolling out the revamped UX and ephemeral sessions :)
